FrameworkSystem/scripts/dirac-admin-proxy-upload.py

```diff
@@ -3,13 +3,23 @@
 # File :    dirac-admin-proxy-upload.py
 # Author :  Adrian Casajus
 ########################################################################
+"""
+Usage:
 
+  dirac-admin-proxy-upload.py (<options>|<cfgFile>)*
+
+Example::
```

Comment:
Here there are two columns while in `Usage` there is only one. I always get confused by this. @andresailer can you teach us again please ?

Reply:
`colons` 

In pure `rst` there should be two colons. In the help message there should be just one, the help message parser will add a second colon so that the rst syntax is respected.

https://github.com/DIRACGrid/DIRAC/blob/e4a1302fb5ff7b3cd21e2eac28e8b75a19d6c918/docs/diracdoctools/cmd/commandReference.py#L204